cishouseholds/pipeline/run.py

```diff
@@ -0,0 +1,28 @@
+import os
+
+import yaml
+
+from cishouseholds.pipeline.bloods_delta_ETL import bloods_delta_ETL  # noqa F401
+from cishouseholds.pipeline.sample_delta_ETL import sample_delta_ETL  # noqa F401
+from cishouseholds.pipeline.survey_responses_version_2_ETL import survey_responses_version_2_ETL  # noqa F401
+from cishouseholds.pipeline.test_ETL import test_ETL  # noqa F401
+
+
+def run_from_config(config_location: str):
+    with open(config_location) as fh:
+        read_data = yaml.load(fh, Loader=yaml.FullLoader)
+    print("Data: ", read_data)
+    if type(read_data) != list:
+        read_data = [read_data]
+    for ETL in read_data:
+        if ETL["run"]:
+            for loc in globals():
+                print("local: ", loc)
+            output_df = globals()[ETL["function"]](ETL["resource_path"])
+            print("OUTPUT:::")
+            output_df.show()
+            print("output location", os.path.join(ETL["output_path"], "output"))
+            output_df.toPandas().to_csv(os.path.join(ETL["output_path"], "output.csv"), index=False)
+
+
+# run_from_config(os.path.join(os.getcwd(), "cishouseholds/pipeline/config.yaml"))
```

Comment:
We could store the location of the config file in a `CISHOUSEHOLDS_CONFIG` environment variable, so that it doesn't need to be hard coded?